### DISCUSSION TITLE
How to do `extern "C"` in CPP2?

### DISCUSSION THREAD

**Discussion by jamadagni** [hsutter/cppfront#1244] [Q&A]
Given that the very spelling of `extern "C"` doesn't seem to fit within CPP2 syntax, how would one go about programming a C API in CPP2?

#624 mentions a metafunction `@c_api` but I guess that's only hypothetical?

**Comment by filipsajdak** [comment]
You can always use mixed mode and mix cpp1 (with `extern "C"`) with cpp2 code.
